css/blocks/projects.css

```diff
@@ -0,0 +1,51 @@
+
+
+.project__list {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 30px;
+  margin: 0;
+  padding: 0px;
+  list-style: none;
+}
+
+.project__list-item {
+  width: 370px;
+  height: 404px;
+  outline: 1px solid rgba(238, 238, 238, 1);
+  background:rgba(238, 238, 238, 1);
```

Comment:
Just this one change - in the mockup it's stated as this color, but actually this part needs to be white (or just no background)
```suggestion
```